spec/file_database_spec.rb

```diff
@@ -65,8 +65,54 @@
     end
   end
 
-  def create_json_contact
-    JSON.generate([test_details])
+  describe '#contact_at' do
+    it 'takes an index and returns the contact in that index' do
```

Comment:
What happens if there is nothing at that index? 
Will the database explode or will it handle it gracefully?

Same for the `update` and `delete` tests.

Always add tests for both the "happy path" and the "not so happy path".